Sources/NIOSSL/TLSConfiguration.swift

```diff
@@ -317,7 +321,8 @@ public struct TLSConfiguration {
                  shutdownTimeout: TimeAmount,
                  keyLogCallback: NIOSSLKeyLogCallback?,
                  renegotiationSupport: NIORenegotiationSupport,
-                 additionalTrustRoots: [NIOSSLAdditionalTrustRoots]) {
+                 additionalTrustRoots: [NIOSSLAdditionalTrustRoots],
+                 sendCANameList: Bool = true) {
```

Comment:
I don't think we should default this here either.

Reply:
If we remove the default here on the private constructor I would need to add a default value to all of the deprecated constructors, correct?  If I changed the default value to `false` here then the both the deprecated and new constructors could take advantage of overriding this property in the same way.  For example:
```swift
var client = TLSConfiguration.forClient(...)
client.sendCANameList = true
        
var client = TLSConfiguration.makeClientConfiguration()
client.sendCANameList = true
```
Thoughts?

Reply:
Oh, good note. In that case, change the default value to `false` here.

Reply:
Resolved in the latest commit.